shopelectro/management/commands/test_db.py

```diff
@@ -9,13 +9,13 @@
 - now you have json file, that'll be used by our TDD tests
 """
 import os
-
+import shutil
 from django.conf import settings
```

Comment:
Please, add a line before third-party packages imports

Reply:
TODO

Reply:
@duker33 the comment above is still not resolved